### ISSUE TITLE
Loading CreateLexer function failed.

### ISSUE THREAD

**Issue by louagej** [BdR76/CSVLint#31]
After installing the latest Notepad++, I receive an error:

```log
Loading CreateLexer function failed.
CSVLint.dll is not compatible with the current version of Notepad++.
Do you want to remove this plugin from the plugins directory to prevent this message from the next launch?

Yes    |     No
```

![220706_6s](https://user-images.githubusercontent.com/7229592/177538529-912f3fe0-e870-418e-8da7-1c3547412843.png)

Current version:
![image](https://user-images.githubusercontent.com/7229592/177538365-1889b4ce-e3aa-4c9e-af2d-110a15f01545.png)

**Comment by BdR76** [comment]
This was a known issue in an earlier version, due to the Lexer v4/v5 update. In the latest version it should be fixed, also see issue #24

Can you install Notepad++ 8.4.4 and try again with the CSV Lint 0.4.5.2 and confirm that it works?

**Comment by louagej** [comment]
@BdR76 ,
thanks, the plugin is working again.
